infrastructure/code/snyk_code_http_client.go

```diff
@@ -539,6 +539,12 @@ func (s *SnykCodeHTTPClient) autofixRequestBody(options *AutofixOptions) ([]byte
 			LineNum:  options.issue.Range.Start.Line + 1,
 		},
 		AnalysisContext: newCodeRequestContext(),
+		IdeExtensionDetails: AutofixIdeExtensionDetails{
+			IdeName:          config.CurrentConfig().IdeName(),
+			IdeVersion:       config.CurrentConfig().IdeVersion(),
+			ExtensionName:    config.CurrentConfig().IntegrationName(),
+			ExtensionVersion: config.CurrentConfig().IntegrationVersion(),
+		},
```

Comment:
The config should be taken from the SnykCodeHTTPClient `s.c.IdeName()` for example.